app/src/main/res/layout/fragment_counter.xml

```diff
@@ -23,11 +23,60 @@
         android:id="@+id/message"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_marginTop="24dp"
+        android:layout_marginTop="16dp"
         android:textAppearance="@style/TextAppearance.ArchDemo.Title3"
         tools:text="Some message here"
         app:layout_constraintTop_toBottomOf="@id/title"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent" />
 
+    <TextView
+        android:id="@+id/click_count_title"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="48dp"
+        android:textAppearance="@style/TextAppearance.ArchDemo.Regular"
+        android:text="Click count: "
+        app:layout_constraintHorizontal_chainStyle="packed"
+        app:layout_constraintTop_toBottomOf="@id/message"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintEnd_toStartOf="@id/click_count" />
+
+    <TextView
+        android:id="@+id/click_count"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:textAppearance="@style/TextAppearance.ArchDemo.Regular.Bold"
+        tools:text="0"
+        app:layout_constraintBaseline_toBaselineOf="@id/click_count_title"
+        app:layout_constraintStart_toEndOf="@id/click_count_title"
+        app:layout_constraintEnd_toEndOf="parent" />
+
+    <ImageButton
```

Comment:
:ajajaj:
Again, just to try to learn your students better habits, this should be a `MaterialButton` with proper style (.Icon).
Furthermore, image buttons have to have content description!

Reply:
Designs and styling are out-of-scope of this lecture.